content/docs/reference/cli/oracle/_index.md

```diff
@@ -21,9 +32,10 @@ Run `autoracle --help` to view the options:
 
 | ORACLE SERVER OPTIONS: | Description | Default | Required? |
 |:--|:--|:--|:--|
```

Comment:
Please rename those flags by refering to the README at https://github.com/autonity/autonity-oracle/blob/ue-improvement/README.md

The `oracle` prefix was removed and the `_` seperator is replace by `.`, some flag was renamed as well.

Reply:
Done, resolved in commit 7b5a773d9d7c2590fad714687b3a1fcdea79e704